demos/ripples.html

```diff
@@ -12,7 +12,7 @@
       <svg width="{{width}}" height="{{height}}">
         <rect x="0" y="0" width="{{width}}" height="{{height}}" fill="#1B1B1B" />
 
-        {{#each points.items key="key" as |point|}}
```

Comment:
`key="key"` fails with

<img width="443" alt="screen shot 2016-04-01 at 14 24 19" src="https://cloud.githubusercontent.com/assets/2526/14208077/7625368e-f815-11e5-94fb-8f8c1f62e321.png">

as we `this.items = new Array(500);` which results in key lookups for undefined items

<img width="613" alt="screen shot 2016-04-01 at 14 22 34" src="https://cloud.githubusercontent.com/assets/2526/14208130/ba36d3e6-f815-11e5-8143-8da94303f4bf.png">